spec/models/test_track/misconfiguration_notifier_spec.rb

```diff
@@ -5,27 +5,19 @@
 
   describe "#notify" do
     context "in development environment" do
-      around do |example|
+      it "raises" do
         with_rails_env "development" do
-          example.run
+          expect { subject.notify("something is misconfigured") }.to raise_exception 'something is misconfigured'
```

Comment:
nit: just noticed we're using raise_error above but raise_exception here. not sure if they're different under the covers, but it seems like we only need to be using one of them.